Widgets/BackgroundRenderer.cpp

```diff
@@ -70,7 +70,7 @@ void BackgroundRenderer::paintEvent(QPaintEvent * /* event */) {
     painter.fillRect(QRectF(0, 0, pixmap.width() * zoom, pixmap.height() * zoom), ArtManager::GetTransparenyBrush());
 
     painter.scale(zoom, zoom);
-    bool transparent = model->data(Background::kTransparentFieldNumber).toBool();
+    bool transparent = true;  // deprecated
```

Comment:
Why true? Isn't the default false, or does this mean "image is already transparent"?

Reply:
That's a mistake, thanks for catching it, I did mean false.